cs/build/internal/Common.Internal.targets

```diff
@@ -1,23 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Condition="'$(TargetFrameworkProfile)' != 'Profile78'">
+  <ItemGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true' and '$(TargetFrameworkProfile)' != 'Profile78'">
     <Reference Include="System" />
     <Reference Include="System.Xml" />
   </ItemGroup>
-  <Import Condition="'$(TargetFrameworkProfile)' == 'Profile78'" Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
-  <Import Condition="'$(TargetFrameworkProfile)' != 'Profile78'" Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Condition="'$(UsingMicrosoftNETSdk)' != 'true' and '$(TargetFrameworkProfile)' == 'Profile78'" Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
+  <Import Condition="'$(UsingMicrosoftNETSdk)' != 'true' and '$(TargetFrameworkProfile)' != 'Profile78'" Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="..\nuget\Common.targets"/>
   <Import Project="Versions.targets" />
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true'">
     <DocumentationFile>$(OutDir)\$(TargetName).xml</DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' == 'true'">
```

Comment:
Minor: can merge this into one property group with a condition on the property itself to save two lines.